source/css/sass/_grid.scss

```diff
@@ -12,5 +12,5 @@
 }
 
 .grid--single-column {
-  @include margin(1.6%, "inline");
+  //@include margin(1.6%, "inline");
```

Comment:
Running grid and content header both off of `wrapper` for now. Will need addressing properly.

Reply:
(Not a huge fan of leaving commented-out code, but it doesn't appear in the compiled form.)

Reply:
Me neither. Deliberately leaving this code smell for the short term in so I can't ignore the problem.